portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,92 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Discover China</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Baloo+Tamma+2:wght@700&family=Lato:wght@300&display=swap" rel="stylesheet">
   </head>
+  
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Discover China 🇨🇳</h1>
+      <h2>你好! (Nǐ hǎo!)</h2>
+      <p><b>China</b> (中国), officially the People's Republic of China, is a country in East Asia. </p>
```

Comment:
Prefer using <strong> over <b> tags (and <em> over <i>)
```suggestion
      <p>
        <strong>China</strong> (中国), officially the People's Republic of China, is a country in East Asia.
      </p>
```
https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

Reply:
done :)  (the explanation on stack overflow in regards to accessibility was really cool!)